<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-12T17:44:39 IST" name="Default suite" finished-at="2023-02-12T17:44:58 IST" duration-ms="18945">
    <groups>
    </groups>
    <test started-at="2023-02-12T17:44:39 IST" name="Default test" finished-at="2023-02-12T17:44:58 IST" duration-ms="18945">
      <class name="GithubProject.test.TestCase">
        <test-method is-config="true" signature="setUpExtents()[pri:0, instance:GithubProject.test.TestCase@791f145a]" started-at="2023-02-12T17:44:39 IST" name="setUpExtents" finished-at="2023-02-12T17:44:39 IST" duration-ms="221" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpExtents -->
        <test-method is-config="true" signature="driverRun()[pri:0, instance:GithubProject.test.TestCase@791f145a]" started-at="2023-02-12T17:44:39 IST" name="driverRun" finished-at="2023-02-12T17:44:43 IST" duration-ms="3457" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driverRun -->
        <test-method is-config="true" signature="Instance()[pri:0, instance:GithubProject.test.TestCase@791f145a]" started-at="2023-02-12T17:44:43 IST" name="Instance" finished-at="2023-02-12T17:44:43 IST" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Instance -->
        <test-method signature="login()[pri:1, instance:GithubProject.test.TestCase@791f145a]" started-at="2023-02-12T17:44:43 IST" name="login" finished-at="2023-02-12T17:44:50 IST" duration-ms="6824" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Assertion Failed expected [GitHub1] but found [GitHub]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Assertion Failed expected [GitHub1] but found [GitHub]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at GithubProject.Base.WebDriver.verify(WebDriver.java:78)
at GithubProject.PageImp.LoginPage.EmailandPass(LoginPage.java:30)
at GithubProject.test.TestCase.login(TestCase.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:GithubProject.test.TestCase@791f145a]" started-at="2023-02-12T17:44:50 IST" name="tearDown" finished-at="2023-02-12T17:44:50 IST" duration-ms="395" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=login status=FAILURE method=TestCase.login()[pri:1, instance:GithubProject.test.TestCase@791f145a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="RepoCreate()[pri:2, instance:GithubProject.test.TestCase@791f145a]" started-at="2023-02-12T17:44:50 IST" name="RepoCreate" finished-at="2023-02-12T17:44:55 IST" duration-ms="4700" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RepoCreate -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:GithubProject.test.TestCase@791f145a]" started-at="2023-02-12T17:44:55 IST" name="tearDown" finished-at="2023-02-12T17:44:55 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=RepoCreate status=SUCCESS method=TestCase.RepoCreate()[pri:2, instance:GithubProject.test.TestCase@791f145a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="deleteRepo()[pri:3, instance:GithubProject.test.TestCase@791f145a]" started-at="2023-02-12T17:44:55 IST" name="deleteRepo" finished-at="2023-02-12T17:44:57 IST" duration-ms="2750" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteRepo -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:GithubProject.test.TestCase@791f145a]" started-at="2023-02-12T17:44:57 IST" name="tearDown" finished-at="2023-02-12T17:44:57 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=deleteRepo status=SUCCESS method=TestCase.deleteRepo()[pri:3, instance:GithubProject.test.TestCase@791f145a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="flush()[pri:0, instance:GithubProject.test.TestCase@791f145a]" started-at="2023-02-12T17:44:57 IST" name="flush" finished-at="2023-02-12T17:44:58 IST" duration-ms="520" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flush -->
      </class> <!-- GithubProject.test.TestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
